packages/styles/scss/components/footer/index.scss

```diff
@@ -5,16 +5,14 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import 'carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
-@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/mini-unit';
-@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/breakpoint';
+@import '../../globals/imports';
+//@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/mini-unit';
```

Comment:
@photodow I wanted to get your feedback on this one. I created a global imports file, which seemed to align with the other components but noticed that yours is only targeting specific elements in `@carbon/layout`, was wondering if this would be a deal breaker if we switch this out to the global import instead? I commented out for now and didn't remove.
cc: @kennylam @annawen1

Reply:
I checked and all layout imports are functions and shouldn't affect file size output.

Reply:
@jeffchew @photodow James has suggested in other PRs using `mini-unit` for spacing certain elements, as opposed to some of the `$spacing` tokens. Could go into global.

Reply:
So if I understand you correctly you're talking about including something like `@carbon/layout/scss/layout` which includes a lot of things along with the `mini-unit`? I don't think that's a deal-breaker. As you mentioned the layouts, in particular, is only functions, so it won't add anything unless we use those provided. As long as the functions being used are represented in global somehow I think it'll be ok. Personal preference is to be more explicit with things like this though, and don't include a bunch of things we don't need/use so trying to be minimal, but as long as it works at the end of day we should be good.😀

Reply:
Are we going to do the same with grid, and type? Those we need to be careful with because they add CSS upon build.

Reply:
It might be worth adding a note in the global include to be careful about including imports that would add css on builds.

Reply:
@photodow I added comments to the global import, let me know if that looks good.

cc: @kennylam @annawen1